src/pages/en/guides/markdown-content.md

```diff
@@ -238,32 +240,65 @@ You can import Markdown and MDX files directly into your Astro files. This gives
 
 You can import one specific page with an `import` statement, or multiple pages with [`Astro.glob()`](/en/guides/imports/#astroglob).
 
-```mdx title="/src/pages/posts/great-post.mdx"
+```astro title="src/pages/index.astro" {3,6}
 ---
-author: Ben
+// Import a single file
+import * as myPost from '../pages/post/my-post.md';
+
+// Import multiple files with Astro.glob
+const posts = await Astro.glob('../pages/post/*.md');
+---
+```
+
+Importing Markdown and MDX files makes their [exported properties](#exported-properties) available in the `.astro` component.
```

Comment:
Is there a reason this is not part of the “Exported properties” section below? There’s a lot _shown_ in these code examples with very little explanation about what is happening. I wonder if a restructure to something like this would make sense?

```md
### Exported properties

Brief summary, e.g. “Importing Markdown and MDX files makes their exported…”

#### Exported properties in Markdown

Code samples with a bit of blurb

#### Exported properties in MDX

Code samples with a bit of blurb

#### All exported properties

- Existing list
- 
```

OR alternate idea

```md
Brief summary, e.g. “Importing Markdown and MDX files makes their exported…”

### Using imported Markdown

Code samples with a bit of blurb

### Using imported MDX

Code samples with a bit of blurb

### Exported properties

- Existing list
- 
```

Reply:
Wondering if we can improve this if we do end up keeping it. “in the `.astro` component” makes me think we’re talking about a specific Astro component, but I think the point is that this is something you can do in Astro components more generally.

```suggestion
When you import Markdown and MDX files in an Astro component, you get an object containing their [exported properties](#exported-properties).
```